### PR TITLE
Fix errors in the PNTS Draco decoding docs

### PR DESCRIPTION
**Pull request by jsulli**
The PNTS decoding docs reference the wrong decoder as well as the wrong loader. Spent a bit of time hair pulling and hunting down confusing errors till I realized this.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -170,7 +170,7 @@ scene.add( tilesRenderer2.group );
 
 ## Adding DRACO Decompression Support
 
-Adding support for DRACO decompression within the GLTF files that are transported in B3DM and I3DM formats. The same approach can be used to add support for KTX2 and DDS textures.
+Adding support for DRACO decompression within the GLTF files that are transported in B3DM and I3DM formats. The same approach can be used to add support for KTX2 and DDS textures. Note that GLTF decoding and PNTS decoding require separate decoders.
```
Comment (gkjohnson) [inline README.md:173]: I'm not sure what this added line means. Can it be removed or clarified?

**Review by gkjohnson — COMMENTED** [review]
Thanks! Looks like primarily a typo in the PNTS example snippet.

File: README.md (hunk 1 above)
Comment (jsulli) [reply README.md:173]: In the current docs, the selected draco decoder from unpkg uses the GLTF branch, which *does not work* for compressed PNTS files. This is a really easy detail to miss IMO, I added this line to clarify that. I'll reword things

https://github.com/mrdoob/three.js/tree/dev/examples/jsm/libs/draco
vs
https://github.com/mrdoob/three.js/tree/dev/examples/jsm/libs/draco/gltf

**Comment by jsulli** [comment]
Hopefully this language is clearer

File: README.md (hunk 1 above)
Comment (gkjohnson) [reply README.md]: Oh I see now... how annoying. I'll move this information down to the section below where the PNTS draco loader is being discussed.

**Pull request merged**